python/pyspark/sql/types.py

```diff
@@ -2137,7 +2181,9 @@ def _test() -> None:
     sc = SparkContext("local[4]", "PythonTest")
     globs["sc"] = sc
```

Comment:
`sc` is redundant? I don't find where use it. If it's redundant, better to remove it.

Reply:
Good catch! thanks!